libs/backend/src/scan/cast_vote_records/export.ts

```diff
@@ -383,33 +434,36 @@ async function exportSignatureFileToUsbDrive(
  */
 export async function exportCastVoteRecordsToUsbDrive(
   scannerStore: ScannerStore,
-  usbDrive: UsbDrive,
+  usbDrive: UsbDrive | LegacyUsb,
   resultSheets: ResultSheet[] | Generator<ResultSheet>,
-  exportOptions: ExportOptions = {}
+  exportOptions: ExportOptions
 ): Promise<Result<void, ExportCastVoteRecordToUsbDriveError>> {
   const exportContext: ExportContext = {
     exporter: new Exporter({
       allowedExportPatterns: SCAN_ALLOWED_EXPORT_PATTERNS,
-      getUsbDrives: async () => {
-        const drive = await usbDrive.status();
-        return drive.status === 'mounted' ? [drive] : [];
-      },
+      getUsbDrives:
+        'getUsbDrives' in usbDrive
+          ? usbDrive.getUsbDrives
+          : async () => {
+              const drive = await usbDrive.status();
+              return drive.status === 'mounted' ? [drive] : [];
+            },
```

Comment:
Planning to consolidate on the new `@votingworks/usb-drive` `UsbDrive` interface later (as I promised to during the recent re-architecture check-in)